data/json/npcs/epilogues.json

```
@@ -311,5 +461,155 @@
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_39",
+    "text": "     Joined a gang after your death, she spent months preying on the innocent, before dying in a dispute with a fellow brute."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_40",
+    "text": "     Devoured alive by a group of zombies, she kept thinking of you as she died."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_41",
+    "text": "     She found an abandoned APC and spent years travelling around and collecting personal stories.  She wrote one of the few post-apocalyptic books, detailing the life after the end."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_42",
+    "text": "     Kept wandering for years. Her life was cut short when a group of raiders used her as a target practice."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_43",
+    "text": "     She became a skilled game hunter and one of the best shots in the wasteland.  Years later she joined the Old Guard and was given the position of Marshal."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_44",
+    "text": "     Desperate to save humanity from extinction, she organised her own group of survivors, whose job was to hunt bandits and decorate roads with their mutilated bodies.  She died in a skirmish a few months later."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_45",
+    "text": "     She became an arsonist but burned alive a few weeks later."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_46",
+    "text": "     She kept wandering for a few weeks but eventually died after drinking untreated water."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_47",
+    "text": "     She wandered north and joined a local community there.  She became known for her trapping skills and ensured that the locals always had fresh meat on their tables."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_48",
+    "text": "     She joined the Free Merchants and led their first caravan to Canada.  She died a few years later of dysentery."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_49",
+    "text": "     She built a boat and sailed east towards Europe, hoping that it was not affected by the apocalypse.  She was never heard from again."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_50",
+    "text": "     She led a doomsday cult for a few years, she was sacrificed to their gods after a particularly terrible harvest."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_51",
+    "text": "     Obsessed with chems, she died of an overdose a few months later."
```

I personally don't think that drugs should be referred to as `chems`, but I know it's a fallout reference, so I don't feel too strongly.
```suggestion
    "text": "     Became obsessed with drugs, and died of an overdose a few months after your death."
```
```suggestion
    "text": "     Became obsessed with chems, and died of an overdose a few months after your death."
```